test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -4364,7 +4364,7 @@ BOOST_AUTO_TEST_CASE(dynamic_array_cleanup)
 			uint[] dynamic;
 			function fill() public {
 				for (uint i = 0; i < 21; ++i)
-					dynamic.push(i + 1);
+					dynamic.push();
```

Comment:
Removed the argument because of https://github.com/ethereum/solidity/issues/7840

Reply:
Isn't that fixed by https://github.com/ethereum/solidity/pull/7842 ?

Reply:
Maybe, I wrote this before :p

Reply:
Changed back to the original, let's see

Reply:
Seems to work - what's with the last one?

Reply:
I mean the apparently "still missing" one you mention in https://github.com/ethereum/solidity/pull/7838#issue-346754227 ? Maybe that works as well now?

Reply:
The last one gives `Unimplemented` error